CMakeLists.txt

```diff
@@ -268,11 +255,12 @@ set(AER_LIBRARIES
 	Threads::Threads
 	${SPDLOG_LIB}
 	${DL_LIB}
-	${MUPARSERX_LIB}
 	${THRUST_DEPENDANT_LIBS})
 
 # Cython build is only enabled if building through scikit-build.
 if(SKBUILD) # Terra Addon build
+	add_muparserx_lib()
```

Comment:
Can we name it:  `find_muparserx()` so to keep consistency with the `find_` semantics CMake already have.

Reply:
Not sure about this one, as we are not using the standard `FindLIB.cmake` mechanism. I didn't want anybody to misbelieve that. But I open to change it, though, I don't have a strong position about it.

Reply:
Ok, let's leave it like this.